includes/class-wc-countries.php

```diff
@@ -908,6 +908,7 @@ public function get_country_locale() {
 						),
 						'state'    => array(
 							'required' => false,
+							'hidden' => true,
```

Comment:
Just a minor coding standard issue... If you can please update those new lines to get aligned, like:

```php
'state' => array(
    'required' => false,
    'hidden'   => true,
),
```